api/models.py

```diff
@@ -179,11 +181,12 @@ class LoggedActivity(Base):
     __tablename__ = 'loggedActivities'
     value = db.Column(db.Integer, nullable=False)
     status = db.Column(db.String, default='pending')
-    approver_id = db.Column(db.String)
     approved_at = db.Column(db.DateTime)
 
+    approver_id = db.Column(db.String)
     user_id = db.Column(db.String, db.ForeignKey('users.uuid'))
     society_id = db.Column(db.String, db.ForeignKey('societies.uuid'))
     activity_id = db.Column(db.String, db.ForeignKey('activities.uuid'))
 
-    activity = db.relationship('Activity', uselist=False)
+    activity = db.relationship(
+        'Activity', backref='logged_activities', uselist=False)
```

Comment:
@LarryWachira Can you give more context on this attribute, `activity`?

Reply:
@georgreen With this relationship. You can easily obtain the activity the logged activity was made for. The assumption is that while logging points, you need to select the activity which society execs should have added on the platform.